bigtable-test/bigtable-internal-test-helper/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>bigtable-test</artifactId>
+        <groupId>com.google.cloud.bigtable</groupId>
+        <version>2.0.0-alpha-1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>bigtable-internal-test-helper</artifactId>
+    <description>
+        Module with helper code for Bigtable unit and integration tests.
+        Not intended to be shared with external users.
+    </description>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```

Comment:
unfortunately bigtable-client-core has to support java7, and since this is a dep of it, you are stuck with java7